text/0233-opentelemetry-sandbox.md

```diff
@@ -0,0 +1,93 @@
+
+## Internal details
+
+### Acceptance criteria
+
+A low barrier to entry would be desired for the sandbox. While the process can be refined based on our experience, my initial proposal for the process is the following:
+
+1. Proposals should be written following the template below and have one Technical Committee (TC) and/or Governance Committee (GC) sponsor, who will regularly provide the TC and GC information about the state of the project.
+2. Once a sponsor is found, the TC and GC will vote on accepting this new project on the Slack channel #opentelemetry-gc-tc.
+    1. After one week, the voting closes automatically, with the proposal being accepted if it has received at least one :thumbs-up: (that of the sponsor, presumably).
+    2. If at least one :thumbs-down: is given, or a TC/GC member has restrictions about the project but hasn’t given a :thumbs-down:, the voting continues until a majority is reached or the restrictions are cleared.
+    3. The voting closes automatically once a simple majority of the TC/GC electorate has chosen one side.
+3. Proponents should abide by OpenTelemetry’s Code of Conduct (currently the same as CNCF’s).
+4. There’s no expectation that small sandbox projects will have regular calls, but there is an expectation that all decisions will be made in public and transparently.
+5. Sandbox projects do NOT have the right to feature OpenTelemetry’s name on their websites.
+
+#### Template
+
+> Project name:
+>
+> Repository name:
+>
+> Problems the project will solve:
+>
+> Motivation for joining the sandbox:
+>
+> Zoom room requested?
+
+##### Example
+
+> Project name: OpenTelemetry Collector Community Distributions
+>
+> Repository name: opentelemetry-collector-distributions
+>
+> Problems the project will solve: The OpenTelemetry Collector Builder allows people to create their own distributions, and while the OpenTelemetry Collector project has no intentions (yet) on hosting other more specialized distributions, some community members are interested in providing those distributions, along with best practices on building and managing such distributions, especially around the CI/CD requirements.
+>
+> Motivation for joining the sandbox: I would love to have more community members to contribute with their own distributions. I would also appreciate broader help in keeping them in sync with the upstream Collector.
+>
+> Zoom room requested? No
+
+### Periodic reports
+
+On the first [GC/TC joint call](https://docs.google.com/document/d/1jylE5uZCKV9mrPw8Qrc5ExGyRVbBdqcbWPwni-hB5dE) of the calendar quarter (Jan, Apr, Jul, Oct), the TC/GC sponsors for the sandbox projects MUST provide an update about their sponsored project. Should the sponsor not be able to join the call, a short written report MUST be provided as part of the meeting notes.
+
+### Exiting the sandbox
+
+Projects can be hosted for at most 12 months and are expected to exit before that, with one of the following outcomes:
```

Comment:
12 Months seems a bit of a hard limit, perhaps rephrasing that once the sandbox project has not been "active" for a certain period of time.

Reply:
For example there is already a sandbox for Web JS exprementation https://github.com/open-telemetry/opentelemetry-sandbox-web-js, and having a general 12 month limit would not work for something like that project.

It's also a little different in that we have automated scripts for merging from the "main" repo's into the sandbox (with all commit history) to avoid creating disconnected forks and explicitly stating that the all successful experiments from the sandbox MUST be contributed back to the main repo's.

Part of this is that we explicitly prepend `sandbox-` to every package, and while there are currently no plans for releasing packages from this sandbox, by having the prefix it more explicitly calls out the non-production nature of anything produced from this repo and any branches that it will have.

Reply:
I'm fine removing this part altogether. I added this based on the feedback from @pyohannes: https://github.com/open-telemetry/oteps/issues/231#issuecomment-1580257623

I like forcing a decision after one year, as it would either show that a community could be formed around the subject of the experiment and will continue working on that once it becomes part of a SIG, or that the experiment completed with a concrete outcome (success, failure). I'm not sure about the web-js sandbox, but I'd be interested in knowing more about its timeline and current state.